.github/workflows/android.yml

```diff
@@ -0,0 +1,54 @@
+name: TGMonet Debug
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - .github/workflows/android.yml
+      - app/**
+      - build.gradle.kts
+      - gradle.properties
+      - gradlew
+      - gradlew.bat
+      - settings.gradle.kts
+  pull_request:
+    paths:
+      - .github/workflows/android.yml
+      - app/**
+      - build.gradle.kts
+      - gradle.properties
+      - gradlew
+      - gradlew.bat
+      - settings.gradle.kts
+
+jobs:
+  build_debug_apk:
+    name: Build TGMonet Debug APK
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+
+      - name: Set up JDK 22
+        uses: actions/setup-java@v4
+        with:
+          java-version: '22'
+          distribution: 'oracle'
+          cache: gradle
+
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+
+      - name: Validate Gradle wrapper
+        uses: gradle/actions/wrapper-validation@v4
+      
+      - name: Build with Gradle
+        id: gradle_build_debug
+        run: ./gradlew assembleDebug
+
+      - name: Upload debug apk
+        uses: actions/upload-artifact@v4
+        with:
+          name: app-debug
+          path: app/build/outputs/apk/debug
```

Comment:
Are u sure that it will work?

Maybe need 
```
with:
          name: app-debug.apk
          path: app/build/outputs/apk/debug/*.apk
```

Reply:
nah, i tested, see: [my fork actions](https://github.com/trindadedev13/Telegram-Monet/actions/runs/10622541247)